__tests__/app.test.js

```diff
@@ -95,9 +98,151 @@ describe("GET /api/articles", () => {
                 })
             })
     })
+    test("status: 200 - should accept topic query, which filters the articles by the topic value specified in the query. If the query is omitted, the endpoint should respond with all articles", () => {
+        return request(app)
+            .get("/api/articles?topic=cats")
+            .expect(200)
+            .then(({ body }) => {
+                const { articles } = body
+
+                expect(articles).toBeInstanceOf(Array)
+                expect(articles).toHaveLength(1)
+
+                articles.forEach((article) => {
+                    expect(article.topic).toBe("cats")
+                })
+            })
+    })
+    test("status: 200 - should accept queries, which filters the articles by the topic and order by ascending or descending a specific column", () => {
+        return request(app)
+            .get("/api/articles?topic=mitch&order=ASC")
```

Comment:
As you've got an order specified it might be nice to have a sortby check here (I can see you've covered this elsewhere but could be a nice addition just for test readability 😄)